step-templates/releasenoteshub-pull-trigger.json

```diff
@@ -0,0 +1,83 @@
+{
+  "Id": "8acee70f-0b6d-43a8-ab16-b832b37e0422",
+  "Name": "ReleaseNotesHub  Trigger",
```

Comment:
Could you include the word `pull` as the name of the file defines that verb e.g. 
`"Name": "ReleaseNotesHub Pull Trigger",`